ApiFirst/AppServices/Validator/CustomerCreateDTOValidator.cs

```diff
@@ -0,0 +1,125 @@
+﻿using AppModels.AppModels;
+using FluentValidation;
+
+namespace AppServices.Validation
+{
+    public class CustomerCreateDTOValidator : AbstractValidator<CreateCustomerDTO>
+    {
+        public CustomerCreateDTOValidator()
+        {
+            RuleFor(x => x.Cpf).
+                NotEmpty().
+                Length(11).
+                Must(IsValidCpf).WithMessage("Please enter a valid CPF, this CPF is not valid");
```

Comment:
Como ser quebra de linha, replicar para todos os lugares que estiverem assim
```suggestion
            RuleFor(x => x.Cpf)
                .NotEmpty()
                .Length(11)
                .Must(IsValidCpf).WithMessage("Please enter a valid CPF, this CPF is not valid");
```